book.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'rentals'
+
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Kindly add the `:rentals` to this `accessor` list.

  ```
  attr_accessor :title, :author, :rentals
  ```

Reply:
Fixed! 🔨